acm/src/main/java/org/literacybridge/acm/store/Committable.java

```diff
@@ -23,17 +25,14 @@
     private CommitState commitState;
     private DeleteState deleteState;
 
-    private CommitListener listener;
+    // remembers whether this is a new Committable that was added to the index between the doCommit() and doAfterCommit() steps
```

Comment:
I don't understand this. It sounds like an error to try to add something during what sounds like the commit process.